frontend/src/pages/projects/screens/detail/workspaces/WorkspaceTableRow.tsx

```diff
@@ -11,11 +11,11 @@ import WorkspaceSizeDetails from './WorkspaceSizeDetails';
 import NotebookImagePackageDetails from '../../../notebook/NotebookImagePackageDetails';
 import WorkspaceStorageBars from './WorkspaceStorageBars';
 
-type NotebookTableRowProps = {
+type WorkspaceTableRowProps = {
```

Comment:
This is something I'd like to fix when we are done -- code should not talk UI concepts -- eg `Workspace` should `Notebook` everywhere in our code. But for the time being though, unification makes the most sense.